ci-operator/step-registry/assisted/common/setup/prepare/assisted-common-setup-prepare-commands.sh

```diff
@@ -196,6 +197,11 @@ cat > run_test_playbook.yaml <<-"EOF"
       ansible.builtin.copy:
         src: "{{ SHARED_DIR }}/platform-conf.sh"
         dest: /root/platform-conf.sh
+    - name: Copy assisted-additional-config file
+      become: true
+      ansible.builtin.copy:
+        src: "{{ SHARED_DIR }}/assisted-additional-config"
```

Comment:
file file always exists because of https://github.com/openshift/release/pull/36224/files#diff-c3d5f997e298a51cf57cf8128467bf4c1be8934332979181f1f6756579efebadR31-R34